app/javascript/packs/shared/navigation/components/NotificationsDropdown.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from "react";
+import { NavDropdown } from "react-bootstrap";
+import { FormattedMessage } from "react-intl";
+import axios from "axios";
+import styled from "styled-components";
+
+import { RECENT_NOTIFICATIONS_PATH } from "../../../app/routes";
+import {
+  MAIN_COLOR_BLUE,
+  WILD_SAND_COLOR,
+  MYSTIC_COLOR
+} from "../../../app/constants/colors";
```

Comment:
does it work if you use absolute path instead of relative path?

from "/app/constants/colors"